receiver/vcenterreceiver/metadata.yaml

```diff
@@ -296,6 +296,24 @@ metrics:
       value_type: int
       aggregation: cumulative
     attributes: []
+  vcenter.vm.memory.swapped:
+    enabled: true
+    description: The portion of memory that is granted to this VM from the host's swap space.
+    unit: MiBy
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.vm.memory.ssdswapped:
```

Comment:
Is this a subset of `swapped`?

Reply:
As always [VMware docs](https://vdc-repo.vmware.com/vmwb-repository/dcr-public/790263bc-bd30-48f1-af12-ed36055d718b/e5f17bfc-ecba-40bf-a04f-376bbb11e811/vim.vm.Summary.QuickStats.html) are a bit "obscure", however, yes it is a subset of the swapped memory.
 - `ssdSwappedMemory` -> "The amount of memory swapped to fast disk device such as SSD, in KB."
 - `swappedMemory -> "The portion of memory, in MB, that is granted to this VM from the host's swap space. This is a sign that there is memory pressure on the host."

Moreover, we cannot check the source code since it is not available to be 100% sure 😕

Reply:
Thanks for the feedback, it makes sense! Addressed in [e390ceb](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16621/commits/e390cebc6e55136cb0016e268b1ae695d1f08fb7)